src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+}
+
+::placeholder {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
You don't need to repeat this rule each time, cause you set it at body selector.
If some of tags don't apply it - use `font-family: inherit` instead

Update all over the document